api/v1alpha1/jindoruntime_types.go

```diff
@@ -150,6 +150,9 @@ type JindoRuntimeSpec struct {
 	// Labels will be added on all the JindoFS pods.
 	// +optional
 	Labels map[string]string `json:"labels,omitempty"`
+
+	// +optional
+	Filelog bool `json:"filelog,omitempty"`
```

Comment:
I like `struct` rather than `boolean` for more possible choices for future. For example we'd like to add log level.

```golang
LogConfig LogConfig `json:"logConfig,omitempty"`
```
Maybe we can refer flink crd.

https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/blob/0310df76d6e2128cd5d2bc51fae4e842d370c463/examples/log_config.yaml#L21

Reply:
this is reasonable, what about
`LogConfig map[string]string `json:"logConfig,omitempty"``
to define some log option to jindoruntime

Reply:
lgtm